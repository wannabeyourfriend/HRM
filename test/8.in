2
12
inbox
copyto 0
inbox
copyto 1
copyfrom 0
outbox
add 1
outbox
copyfrom 1
sub 0
outbox
jumpto 1