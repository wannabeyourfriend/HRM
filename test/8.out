Error on instruction 7